src/tsec.exemptions.json

```diff
@@ -34,6 +34,7 @@
 	],
 	"ban-element-innerhtml-assignments": [
 		"sql/workbench/browser/modal/modal.ts",
+		"sql/base/browser/ui/checkbox/checkbox.ts",
```

Comment:
This isn't something we should lightly be adding stuff to. Letting extensions contribute raw HTML is a security risk (XSS and other types of attacks can easily be done if the user can insert arbitrary HTML).

Instead I would rather you identify the scenario exactly you want to support. For example if you want to have the checkboxes be images instead then have you looked at the Card components? Those are basically doing what you want.

Reply:
Makes sense. I hadn't seen this check up until a couple of weeks, and when I saw that there were quite a few exceptions, I elected to add this one as well, but I should have asked instead. 

I did take a look at the card components, and the radio card group component as well. They seem to be build for a specific use, and of course the radio would not work as a checkbox. 

If it makes more sense, I could build a checkbox card group, similar to the radio card group, allowing us to add images to the card.

Reply:
Yeah, this stuff was brought in with the latest vs code merge. We'll be working to remove these as soon as we can - we just haven't gotten around to it yet. 

@alanrenmsft Thoughts on a checkbox card group? Something like that seems reasonable to me to add, and doing so shouldn't be a huge task since it'd mostly act like the radio card group.

Reply:
the radio card group is necessary because of the keyboard navigation requirements, single tab stop on the whole group and use left/right arrow key to navigate within the group. I don't think checkbox have such requirement, so this can be a component on its own, also I am not really happy with the radio card group, it is not very flexible, so for the new component, we should make it a container and extension author can add other components to it.  Maybe we can extend the divContainer and add a property to it called `Selectable : boolean`.